src/ui/EditorNS/editor.cpp

```diff
@@ -34,6 +34,7 @@ namespace EditorNS
 
     void Editor::fullConstructor(const Theme &theme)
     {
+        m_currentLanguage = LanguageService::getInstance().lookupById("plaintext");
```

Comment:
Last issue: use setLanguage(nullptr) or setLanguage(LanguageService:: ...) here. Needed because setLanguage() does a few things like setting indentation.

You'll also have to move setLanguage(..) to the bottom of the page or we get stuck in the JS event loop because the page isn't loaded yet to handle the C_CMD_SET_LANGUAGE.